staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler.go

```diff
@@ -36,6 +36,9 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// GroupVersions from the default source take precedence over all others
+var DefaultSource = "default"
```

Comment:
I suggest a different system.

```
type Source uint

const (
  BuiltInCRDSource Source = 100
  AggregatedAPISource Source = 200
)
```

And instead of making one group special in terms of precedence, use the rule that the smallest number wins.

Or something like this.

Reply:
added this. also fixes the nondeterminism in tiebreaking non-default sources